test/follow.js

```diff
@@ -146,101 +146,6 @@ test('Catchup events', function(t) {
   })
 })
 
-test('Data due on a paused feed', function(t) {
```

Comment:
Even if we don't have these unreliable tests that do loads of timing assertions that lead to flakiness it would be good to maintain coverage of the "pause" of a feed, to ensure that function isn't regressed.

Reply:
Added e3e0125.